subsys/bluetooth/crypto/bt_crypto.c

```diff
@@ -17,7 +21,36 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_crypto);
 
+#if defined(CONFIG_BT_USE_PSA_API)
+int bt_crypto_aes_cmac(const uint8_t *key, const uint8_t *in, size_t len, uint8_t *out)
```

Comment:
could you instead add two new files with `bt_crypto_aes_cmac` implementations for psa and for tinycrypt?